tests/xtro-sharpie/watchOS-MediaPlayer.ignore

```diff
@@ -0,0 +1,2 @@
+# added to expose fields, similar to what is found in mac os x.
+!unknown-type! MPMediaItem bound
```

Comment:
it should not be picked up by **xtro** if it's not registered (and it should not be)

Reply:
API Diff shows it's an `NSObject` while, from your description, it should be a `[Static]` type
```
	// constructors
	public MPMediaItem ();
	public MPMediaItem (Foundation.NSCoder coder);
	protected MPMediaItem (Foundation.NSObjectFlag t);
	protected MPMediaItem (IntPtr handle);
```